.github/workflows/ci-java.yml

```diff
@@ -57,10 +72,13 @@ jobs:
   mvn-verify:
     runs-on: codebuild-bfd-non-prod-platform-large-${{ github.run_id }}-${{ github.run_attempt }}
     needs: workflow
-    if: needs.workflow.outputs.files
+    if: ${{ needs.workflow.outputs.files || (github.actor != 'actions-user' && github.ref_name == 'master') }}
```

Comment:
This *should* ensure that release-oriented `push` events don't unnecessarily trigger the CI.